package.json

```diff
@@ -49,10 +49,22 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "eb:prod": "npm run build && npm run start:prod",
-    "prettier": "prettier --write \"./src/**/*.{js,jsx}\""
+    "prettier": "prettier --write \"./src/**/*.{js,jsx}\"",
+    "storybook": "start-storybook -p 6006 -s storybook-static",
```

Comment:
this is fine too, but if you just add  `start-storybook -p 6006` it will default to `storybook-static`.